### PR TITLE
Create Gamedesign Handheld

### PR DESCRIPTION
**Pull request by jefiedler**
First draft of our gameFishBoy. 

Screenshot 2020-08-03 at 15 00 08

Our created design: 

https://xd.adobe.com/view/e67c4a9c-374c-460e-886e-bf1988c267ac-b32f/specs/

Screenshot 2020-07-31 at 18 23 28

### CODE REVIEW

**Comment by rheimers** [comment]
> First draft of our gameFishBoy.
> 
> Screenshot 2020-08-03 at 15 00 08
> 
> Our created design:
> 
> https://xd.adobe.com/view/e67c4a9c-374c-460e-886e-bf1988c267ac-b32f/specs/
> 
> Screenshot 2020-07-31 at 18 23 28

Open issues: 
- Bring buttons to action
- Implement game

File: src/components/handheldDesign/handheldDesign.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  left: 647px;
+  width: 163px;
+  height: 162px;
+  justify-self: center;
+  margin: 0 40px 30px 0;
+}
+
+.joystick {
+  /* grid-column: 2 / 3;
+  grid-row: 1 / 2; */
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.css:54]:
```suggestion
```

File: src/components/handheldDesign/handheldDesign.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  grid-row: 2 / 3;
+  align-self: end;
+  justify-self: end;
+  margin: 0 40px 60px 0;
+}
+
+/* .gameBody__display--game {
+  height: 100px;
+  width: 100px;
+} */
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.css:95]:
```suggestion
```

File: src/components/handheldDesign/handheldDesign.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { createElement } from "../../utils/elements";
+import starBtnImg from "../../assets/imgHandheldDesign/starButton.svg";
+import joyBtnImg from "../../assets/imgHandheldDesign/joystick.svg";
+import aBtnImg from "../../assets/imgHandheldDesign/aButton.svg";
+import bBtnImg from "../../assets/imgHandheldDesign/bButton.svg";
+import speakImg from "../../assets/imgHandheldDesign/speaker.svg";
+// import { basic } from "../game/game.stories";
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.js:7]:
```suggestion
```

File: src/components/handheldDesign/handheldDesign.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+export const createHandheldBody = () => {
+  const body = createElement("div", { className: "gameBody" });
+  const display = createElement("section", { className: "gameBody__display" });
+
+  // const game = basic();
+  // const gameScreen = createElement("div", {
+  //   className: "gameBody__display--game",
+  // });
+  // gameScreen.append(game);
+  // display.append(gameScreen);
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.js:18]:
```suggestion
```

File: src/components/handheldDesign/handheldDesign.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  const cockpitImages = [
+    { imgSrc: starBtnImg, imgClass: "starButton" },
+    { imgSrc: joyBtnImg, imgClass: "joystick" },
+    { imgSrc: aBtnImg, imgClass: "aButton" },
+    { imgSrc: bBtnImg, imgClass: "bButton" },
+    { imgSrc: speakImg, imgClass: "speaker" },
+  ];
+
+  cockpitImages.forEach((controlImg) => {
+    const button = createElement("img", { src: controlImg.imgSrc });
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.js:30]: Would add an alt attribute/key-value-pair for the images in the object behind src:

File: src/components/handheldDesign/handheldDesign.js (hunk 4 above)
Comment (manuelpapa) [inline src/components/handheldDesign/handheldDesign.js:18]: not necessary anymore

**Review by manuelpapa — COMMENTED** [review]
<3 gw!

File: src/components/handheldDesign/handheldDesign.js (hunk 4 above)
Comment (DrBennemin) [inline src/components/handheldDesign/handheldDesign.js:18]: Please delete commented lines <3

File: src/components/handheldDesign/handheldDesign.js (hunk 3 above)
Comment (DrBennemin) [inline src/components/handheldDesign/handheldDesign.js:7]: Please delete commented lines <3

File: src/components/handheldDesign/handheldDesign.css (hunk 2 above)
Comment (DrBennemin) [inline src/components/handheldDesign/handheldDesign.css:95]: Please delete commented lines <3

File: src/components/handheldDesign/handheldDesign.css (hunk 1 above)
Comment (DrBennemin) [inline src/components/handheldDesign/handheldDesign.css:54]: Please delete commented lines <3

File: src/components/handheldDesign/handheldDesign.css [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  margin: 0 40px 30px 0;
+}
+
+.joystick {
+  /* grid-column: 2 / 3;
+  grid-row: 1 / 2; */
+  grid-area: 1 / 2 2 / 3;
+  top: 589px;
+  left: 787px;
+  width: 156px;
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.css:58]: really necessary? made no difference while deleting it in the inspector.

File: src/components/handheldDesign/handheldDesign.css [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto;
+}
+
+.starButton {
+  grid-column: 1 / 2;
+  grid-row: 2 / 3;
+  top: 722px;
+  left: 647px;
+  width: 163px;
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.css:46]: really necessary? made no difference while deleting it in the inspector.

File: src/components/handheldDesign/handheldDesign.css (hunk 1 above)
Comment (MariusS94) [inline src/components/handheldDesign/handheldDesign.css:54]: Please delete the commented code. Thx. 🙏 😊

**Review by DrBennemin — CHANGES_REQUESTED** [review]
Please delete commented lines <3 
The rest looks pretty neat to me. Nice job 🥳 🎉 🐜

File: src/components/handheldDesign/handheldDesign.css [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+  justify-self: end;
+}
+
+.bButton {
+  grid-column: 4 / 5;
+  grid-row: 1 / 2;
+  top: 663px;
+  left: 1182px;
+  width: 86px;
+  height: 86px;
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.css:80]: really necessary? made no difference while deleting it in the inspector.

File: src/components/handheldDesign/handheldDesign.css [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+  justify-self: end;
+  margin: 30px 0 0 80px;
+}
+
+.aButton {
+  grid-column: 3 / 4;
+  grid-row: 2 / 3;
+  top: 754px;
+  left: 1087px;
+  width: 86px;
```
Comment (slawoe) [inline src/components/handheldDesign/handheldDesign.css:69]: really necessary? made no difference while deleting it in the inspector.

File: src/components/handheldDesign/handheldDesign.css (hunk 2 above)
Comment (MariusS94) [inline src/components/handheldDesign/handheldDesign.css:95]: Please delete the commented code. Thx. 🙏 😊

File: src/components/handheldDesign/handheldDesign.js (hunk 3 above)
Comment (MariusS94) [inline src/components/handheldDesign/handheldDesign.js:7]: Please delete the commented code. Thx. 🙏 😊

File: src/components/handheldDesign/handheldDesign.js (hunk 4 above)
Comment (MariusS94) [inline src/components/handheldDesign/handheldDesign.js:18]: Please delete the commented code. Thx. 🙏 😊

**Review by MariusS94 — CHANGES_REQUESTED** [review]

**Review by AlexRyzhkova — COMMENTED** [review]
lgtm :) 
Please delete comments before merging.

**Review by slawoe — CHANGES_REQUESTED** [review]
good job, but some comments can be deleted and i have tried to manipulate your css code in the inspector (height. width, etc. of your buttons and in many cases, nothing changed -> so is it really necessary to keep it in your css?

**Pull request merged**